controllers/useraccount/useraccount_controller_test.go

```diff
@@ -492,6 +501,21 @@ func TestReconcile(t *testing.T) {
 			assert.Contains(t, err.Error(), "unable to update status")
 		})
 
+		t.Run("no nstemplateset to create", func(t *testing.T) {
```

Comment:
do we have a test also for updates?
I mean - NSTemplateSet exists in the cluster, but UserAccount has the NSTemplateSet parameter set to nil.

Reply:
I've added a test in 55cf7e0 but honestly I'm not sure it's going to address your comment 🤷‍♂️